app/scripts/templates/settings/change_password.mustache

```diff
@@ -23,7 +23,7 @@
           {{#t}}Show{{/t}}
         </label>
 
-        <div class="input-help links"><a href="/reset_password" class="reset-password">{{#t}}Forgot password?{{/t}}</a></div>
+        <div class="input-help-forgot-pw links"><a href="/reset_password" class="reset-password">{{#t}}Forgot password?{{/t}}</a></div>
```

Comment:
@vbudhram Hey, yeah this changes the link location: after/before: ![](http://v14d.com/i/574496787ce42.jpg)